server_src/runners/state.js

```diff
@@ -2546,23 +2546,23 @@ class Runner {
     return ClientAPI.getRetry(
       farmPrinters[index].printerURL,
       farmPrinters[index].apikey,
-      "api/plugin/pluginmanager"
+      "plugin/pluginmanager/plugins"
     )
       .then((res) => {
         return res.json();
       })
-      .then((res) => {
-        farmPrinters[index].pluginsList = res.repository.plugins;
```

Comment:
@NotExpectedYet did you intend to call the `repository.plugins` here? I'd expect the installed plugins instead of the available ones.

Reply:
I grab the available ones for the plugin installer.